packages/functions/package.json

```diff
@@ -43,7 +43,7 @@
   "scripts": {
     "pretest": "pnpm run build:code",
     "test": "vitest",
-    "build": "pnpm run build:code && pnpm run build:docs",
+    "build": "pnpm run build:code",
```

Comment:
The side-effect of this produces a `docs` folder that is not gitignored, meaning this build runs in an infinite loop. I kind of expected turbo to have some sort of "watch" ignore pattern but here's their [official recommendation](https://turbo.build/repo/docs/reference/watch#task-outputs):

> If you have tasks that write to files checked into source control, there is a possibility that Watch Mode will run in an infinite loop. This is because Watch Mode watches your files for changes and will re-run tasks in packages that have changed. If a task creates a change, then that will trigger the task again.

> Watch Mode has some logic to prevent this from happening using file hashes, but it isn't foolproof. To avoid this issue, we recommend removing any task outputs from git.

So not sure how we're using these outputs, can I just add `docs` to the `.gitignore`?